Braintree/Drop-In/BTDropInViewController.h

```diff
@@ -86,13 +93,45 @@
 /// @see callToActionAmount
 @property (nonatomic, assign) BOOL shouldHideCallToAction;
 
-/// Fetches the customer's saved payment methods and populates Drop In with them.
+#pragma mark - Card Form
+
+/// The card number.
 ///
-/// @note For the best user experience, you should call this method as early as
-///       possible (after initializing BTDropInViewController, before presenting it)
-///       in order to avoid a loading spinner.
-- (void)fetchPaymentMethods;
+/// If you set a card number longer than is allowed by the card type,
+/// it will not be set.
```

Comment:
This comment is kind of indirect in describing what this property actually does.